### PR TITLE
Add rule for scumware.org

### PR DESCRIPTION
**Pull request by wesinator**

### CODE REVIEW

File: src/chrome/content/rules/scumware.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+          scumware.org : certificate is for home.pl
+-->
+<ruleset name="Scumware.org">
+        <target host="scumware.org" />
+        <target host="www.scumware.org" />
+
+        <rule from="^http://scumware\.org/"
+                to="https://www.scumware.org/" />
+        
+        <rule from="^http:" to="https:" />
```
Comment (pipboy96) [inline src/chrome/content/rules/scumware.xml]: Please replace 8 spaces with a single tab.

File: src/chrome/content/rules/scumware.xml [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
-
-        <rule from="^http://scumware\.org/"
-                to="https://www.scumware.org/" />
-        
-        <rule from="^http:" to="https:" />
+	<target host="scumware.org" />
+	<target host="www.scumware.org" />
+	
+	<rule from="^http://scumware\.org/"
+		  to="https://www.scumware.org/" />
```
Comment (pipboy96) [inline src/chrome/content/rules/scumware.xml]: Please do not mix spaces and tabs.

File: src/chrome/content/rules/scumware.xml [hunk 3]
```diff
@@ -3,11 +3,11 @@
           scumware.org : certificate is for home.pl
```
Comment (pipboy96) [inline src/chrome/content/rules/scumware.xml]: This applies to comments too.

File: src/chrome/content/rules/scumware.xml (hunk 2 above)
Comment (wesinator) [reply src/chrome/content/rules/scumware.xml]: sorry, github online editing

**Comment by pipboy96** [comment]
www.scumware.org has incomplete certificate chain: https://whatsmychaincert.com/?www.scumware.org, all targets are broken.

**Pull request closed without merging**